sfcasts/install.md

```diff
@@ -1,37 +1,38 @@
 # Install
 
-Hey friends! We are in for a treat with this tutorial! It's EasyAdmin, my
-favorite admin generated for Symfony. It just gives you so many features out of
-the box and it looks great. This shouldn't really be a surprise because its creator is
-the always-impressive Javier Aguila. So let's have some fun and learn how to bend
-EasyAdmin to our will, because getting a lot of features for free is great, as long
-as we can extend everything when we need to. To get the most “easy” out of 
-EasyAdmin, you should definitely code along with me. 
+Hey friends! We are in for a treat with this tutorial! It's EasyAdmin, my favorite admin generated for Symfony. It just gives you so many features out of the box and it looks great. This shouldn't really be a surprise because its creator is the always-impressive Javier Aguila. So let's have some fun and learn how to bend EasyAdmin to our will, because getting a lot of features for free is great, as long as we can extend everything when we need to. To get the most “easy” out of  EasyAdmin, you should definitely code along with me. 
 
 You probably know the drill: Download the course code from this page and unzip it to find a start directory with the same code that you see here. Check out the `README.md` file for all these setup goodies. I've already done all of these steps except for two. 
```

Comment:
`start/` 

with "ticks" around it. This is a tough one because you don't see it on the screen, but there is literally a directory called "start". And I like to always put directory names with a closing `/` to make it more obvious that it's a directory.